src/components/Accordion/styles/LabelText.js

```diff
@@ -3,5 +3,6 @@ import styled from 'styled-components';
 // FIXME: it would be nice to craft the right CSS so that this element is unecessary.
 export default styled.span`
   margin: auto;
+  padding-bottom: 1px;
```

Comment:
If you have to do hacky stuff like a `1px` `padding-bottom`, that may mean that the `display` property isn't set correctly on this element.  Possibly try `display: inline-block` to see if that fixes the layout of the box model versus forcing it with padding.

Reply:
After a F2F conversation, this comment may not be 100% relevant.  Dismiss if that's the case  👍